server/src/pages/settings/controller.js

```diff
@@ -18,6 +18,7 @@ exports.disconnectDevice = function() {
     if (req.status >= 300) {
       // FIXME: a lame way to do an error message
       window.alert("Error disconnecting: " + req.status + " " + req.statusText);
+      window.Raven.captureException(new Error("Error disconnecting"));
```

Comment:
This should have some more information. Probably it would be enough to have

    new Error(`Error disconnecting: ${req.status} ${req.statusText}`)